src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -73,7 +75,9 @@
 				<loan-card-controller
 					v-for="loan in loans"
 					:items-in-basket="null"
-					:is-visitor="true"
+					:is-visitor="userId === null"
+					:is-logged-in="userId !== null"
+					:user-id="userId !== null ? userId.toString() : null"
```

Comment:
These additional fields on loan card controller contribute to showing the correct loan button state if a lender has already lent to the loan.